HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -2855,9 +2856,16 @@ def to_oga(doc)
         efficiency_value = @cooling_efficiency_seer
         efficiency_value_isdefaulted = @cooling_efficiency_seer_isdefaulted
       elsif [HVACTypeRoomAirConditioner].include? @cooling_system_type
-        efficiency_units = UnitsEER
-        efficiency_value = @cooling_efficiency_eer
-        efficiency_value_isdefaulted = @cooling_efficiency_eer_isdefaulted
+        if not @cooling_efficiency_eer.nil?
+          efficiency_units = UnitsEER
+          efficiency_value = @cooling_efficiency_eer
+          efficiency_value_isdefaulted = @cooling_efficiency_eer_isdefaulted
+        else
+          # Fixme: In this way if efficiency is specified in EER, CEER will not show up in the xml generated(in.xml), even though in the simulation we're using ceer being converted to.
```

Comment:
Indeed the EER->CEER conversion is not a "default" but rather a correlation we use for modeling. So I suggest:
1. Revert the code in hpxml_defaults.rb and implement the EER->CEER conversion in hvac.rb's apply_room_ac (or whatever it's called) method.
2. Change the code above to `elsif not @cooling_efficiency_ceer.nil?`

We've taken similar approach for water heaters (EF vs UEF) and appliances (EF vs CEF, MEF vs IMEF, etc.).